base_location/models/company.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Nicolas Bessi, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    city_id = fields.Many2one(
+        'res.city',
+        compute='_compute_address',
+        inverse='_inverse_city_id',
+        string="City"
+    )
+    zip_id = fields.Many2one(
+        'res.better.zip',
+        string='ZIP Location',
+        compute='_compute_address',
+        inverse='_inverse_zip_id',
+        oldname="better_zip_id",
+        help='Use the city name or the zip code to search the location',
+    )
+    '''
+        In order to keep the same logic used in odoo, fields must be computed
```

Comment:
@pedrobaeza Comments here. I think that we can merge now.

Reply:
This is not comment, but docstring, and can't be in the middle of the code. You should make here a real comment:
```
# In order to keep the same logic used in odoo, fields must be computed
# and inversed, not related. This way, we can ensure that it works
# ...
```